packages/react-native/src/primitives/Label/utils.ts

```diff
@@ -0,0 +1,13 @@
+import { ViewStyle } from 'react-native';
+import { LabelPosition } from './types';
+
+const FLEX_DIRECTIONS: Record<LabelPosition, ViewStyle['flexDirection']> = {
+  start: 'row-reverse',
+  end: 'row',
+  top: 'column-reverse',
+  bottom: 'column',
+};
+
```

Comment:
```suggestion
 /**
 * The `flexDirection` value returned from `getFlexDirectionFromLabelPosition`
 * assumes a `Label` component follows the `Primary` component it labels inside
 * the `Container` that the `flexDirection` is applied to.
 *
 * Example Usage:
 * ```jsx
 *   const flexDirection = getFlexirectionFromLabelPostiion(labelPosition);
 *   <Container style={{ flexDirection }}>
 *     <Primary />
 *     <Label />
 *   </Container>
 * ```
 */
```